shotgun_api3/lib/xmlrpclib.py

```diff
@@ -289,40 +290,60 @@ def __repr__(self):
 
 if _bool_is_builtin:
     boolean = Boolean = bool
-    # to avoid breaking code which references xmlrpclib.{True,False}
-    True, False = True, False
+    try:
+        # TODO: remove this when py2 support is completely dropped
+        # to avoid breaking code which references xmlrpclib.{True,False}
+        True, False = True, False
+
+    except SyntaxError:
```

Comment:
Not sure what this `True/False` trickery is about, in py3 assignment to keyword is not allowed any more.

This should have the code behave like it used in py2, and won't fail to parse with py3, however you may need to re-examine what to do in py3 in this case.

Ie: this code is py2 equivalent, but no guarantees it'll do what you want in py3